prototyping/dscp-lang-flows/hyproof-token-flows.json

```diff
@@ -0,0 +1,288 @@
+[
```

Comment:
move to root folder as `processFlows.json` please. Also make sure this is generated off of the latest version of `dscp-lang` on `dscp-node/main`

Reply:
OK .. FIY , i am witnessing some strange messages when deploying this with the process manager

```
Unable to map [u8; 32] to a lookup index
```

Reply:
I'm going to add a spike to the backlog for this. It doesn't appear to affect anything (tests still pass) but I noticed this in the CI release run for matchmaker aswell.